tutorial/example_playbook.yaml

```diff
@@ -0,0 +1,12 @@
+---
+- name: Add Cloud Ops Agent to hosts
+  hosts: all
+  become: true
+  roles:
+    - role: google-cloud-ops-agents-ansible
+      vars:
+        agent_type: monitoring
```

Comment:
Is there any special reason we need to use the legacy monitoring and logging agents in the example? Since we are promoting the Ops Agent, we probably wanna use `agent_type: ops-agent` instead, right? In other words:

Replace

```
  - role: google-cloud-ops-agents-ansible
      vars:
        agent_type: monitoring

    - role: google-cloud-ops-agents-ansible
      vars:
        agent_type: logging
```

With

```
  - role: google-cloud-ops-agents-ansible
      vars:
        agent_type: ops-agent
```